app/src/main/java/ch/epfl/sdp/entity/ShelterArea.java

```diff
@@ -6,6 +6,7 @@
 import ch.epfl.sdp.geometry.GeoPoint;
 import ch.epfl.sdp.map.Displayable;
 import ch.epfl.sdp.map.MapApi;
+import ch.epfl.sdp.utils.RandomGenerator;
```

Comment:
Are we use RandomGenerator here?

Reply:
No, it's a relic of some function which I deleted. Thank you for pointing it out